File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/telefon/EndringTelefon.java
package no.nav.personopplysninger.features.endreopplysninger.domain.telefon;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import io.swagger.annotations.ApiModel;
import no.nav.personopplysninger.features.endreopplysninger.domain.Endring;
import no.nav.personopplysninger.features.endreopplysninger.domain.telefon.Telefonnummer;

@ApiModel(description = "Responsen fra GET /api/v1/endring/{endringId} når endring gjelder telefonnummer.")
@JsonIgnoreProperties(ignoreUnknown = true)
public class EndringTelefon extends Endring<EndringTelefon> {
    private Telefonnummer innmeldtEndring;

    public Telefonnummer getInnmeldtEndring() {
        return innmeldtEndring;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysninger/domain/telefon/EndringTelefon.kt
package no.nav.personopplysninger.features.endreopplysninger.domain.telefon

import no.nav.personopplysninger.features.endreopplysninger.domain.Endring

class EndringTelefon : Endring<EndringTelefon> {
    var innmeldtEndring: Telefonnummer? = null

    constructor()

    constructor(innmeldtEndring: Telefonnummer) {
        this.innmeldtEndring = innmeldtEndring
    }
}
